Update mcc-isp-signup.md (#11602)
MicrosoftDocs/windows-itpro-docs: windows/deployment/do/mcc-isp-signup.md

1. **Azure Pay-As-You-Go subscription**: Microsoft Connected Cache is a completely free-of-charge service hosted in Azure. You'll need to have a Pay-As-You-Go subscription in order to onboard to our service. To create a subscription, go to the [Pay-As-You-Go subscription page](https://azure.microsoft.com/offers/ms-azr-0003p/).

> [!NOTE]

> > Microsoft Connected Cache is a completely free service for operators. None of the resources created in Azure will incure any charges.
> > Please be aware of any additional services that may be selected as part of the Azure sign up process.

Reviewer: ```suggestion
> - Microsoft Connected Cache is a completely free service for operators. None of the resources created in Azure will incur any charges.
> - Be aware, however,  that any additional services that might be selected as part of the Azure sign-up process might incur charges.
```
